ledger/src/blockstore_metric_report_service.rs

```diff
@@ -34,6 +34,7 @@ impl BlockstoreMetricReportService {
                     BLOCKSTORE_METRICS_REPORT_PERIOD_MILLIS,
                 ));
                 blockstore.submit_rocksdb_cf_metrics_for_all_cfs();
+                blockstore.report_rpc_api_metrics();
```

Comment:
The existing period is 10s; figured that seems roughly good enough here but open to opinions if we think this should go more granular. This should already be a substantial decrease in the number of datapoints generated, so don't think using a smaller number like 5s would be a big deal